pkg/proxy/apis/config/fuzzer/fuzzer.go

```diff
@@ -28,13 +28,30 @@ import (
 	"k8s.io/utils/ptr"
 )
 
+// getRandomDualStackICIDR returns a random dual-stack CIDR.
```

Comment:
This is how I did it

https://github.com/kubernetes/kubernetes/blob/6ac20677c781e72346ccc56daafb161d6750cbbd/pkg/apis/networking/fuzzer/fuzzer.go#L88-L136

Reply:
Should I move `generateRandomIP` and `generateRandomCIDR` to `k/utils/net` and then refer to that in `pkg/apis/networking/fuzzer` and  `pkg/proxy/apis/config/fuzzer`?